icu4c/source/common/uresbund.cpp

```diff
@@ -1146,6 +1146,17 @@ static UResourceBundle *init_resb_result(const ResourceData *rdata, Resource r,
                                     r = res_findResource(&(dataEntry->fData), r, &myPath, &temp);
                                     if(r != RES_BOGUS) { /* found a resource, but it might be an indirection */
                                         resB = init_resb_result(&(dataEntry->fData), r, temp, -1, dataEntry, result, noAlias+1, resB, status);
+                                        if (temp == NULL || uprv_strcmp(keyPath, temp) != 0) {
```

Comment:
optional: nullptr not NULL

Reply:
There are lots of places in the same function where the existing code is using "`NULL`", and I didn't want to change all of them, and so I followed the established precedent.  Did we all decide that one should change "`NULL`" to `"nullptr`" everywhere when making other changes to a file?  If so, should I do that just to the functions I changed, or to the whole file?

Reply:
I like to use nullptr when I write new code, but it creates a mix with the existing NULL. Maybe for 71 we should plan to change all NULL to nullptr in .cpp files?

Anyway, I marked it as "optional", so you can leave it as is.